From 46d1fb6591a87520548dcaec1546f0939d6ecb03 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <sanyog.r.kale@intel.com>
Date: Fri, 21 Apr 2017 12:21:17 +0530
Subject: [PATCH 0588/1132] SoundWire: Select default frame shape based on
 platform

This patch selects default frame shape in SoundWire controller
init based on FPGA/NONFPGA platform.

Change-Id: I07cb3f578367ef3afb4ea1e0db905d562d134f8f
Signed-off-by: Sanyog Kale <sanyog.r.kale@intel.com>
Reviewed-on:
Reviewed-by: Prodduvaka, Leoni
Reviewed-by: Kp, Jeeja <jeeja.kp@intel.com>
Reviewed-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Reviewed-by: R, Dharageswari <dharageswari.r@intel.com>
Tested-by: Avati, Santosh Kumar <santosh.kumar.avati@intel.com>
---
 drivers/sdw/sdw_cnl_priv.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/sdw/sdw_cnl_priv.h b/drivers/sdw/sdw_cnl_priv.h
index 504df88..b7f44e1 100644
--- a/drivers/sdw/sdw_cnl_priv.h
+++ b/drivers/sdw/sdw_cnl_priv.h
@@ -28,12 +28,13 @@
 #define SDW_CNL_CMD_WORD_LEN	4
 #define SDW_CNL_DEFAULT_SSP_INTERVAL	0x18
 #define SDW_CNL_DEFAULT_CLK_DIVIDER	0
-#define SDW_CNL_DEFAULT_FRAME_SHAPE	0x30
 
 #if IS_ENABLED(CONFIG_SND_SOC_INTEL_CNL_FPGA)
 #define SDW_CNL_DEFAULT_SYNC_PERIOD	0x257F
+#define SDW_CNL_DEFAULT_FRAME_SHAPE	0x48
 #else
 #define SDW_CNL_DEFAULT_SYNC_PERIOD	0x176F
+#define SDW_CNL_DEFAULT_FRAME_SHAPE	0x30
 #endif
 
 #define SDW_CNL_PORT_REG_OFFSET		0x80
-- 
2.7.4

